shared/common/build/tools/update_podspec_headers.sh

```diff
@@ -30,6 +30,8 @@
 # set -x
 set -e
 
+VERSION=4.0.1 # XXX: update that version for every release
```

Comment:
Not as such, that I can think of.  `SalesforceSDKConstants.h` is the closest match, but won't help in this context.